### PR TITLE
Update google-symptoms.md

### PR DESCRIPTION
**Pull request by shekelto**

### CODE REVIEW

File: docs/api/covidcast-signals/google-symptoms.md [hunk 1]
```diff
@@ -49,22 +49,9 @@ The `sum_anosmia_ageusia_raw_search` signals are simply the raw sum of the
...
-normalized popularity across the entire published time range for that region
-over all symptoms. The values of symptom popularity are **NOT** comparable
-across geographic regions. Due to the scaling step, most of the values should be
-in the range 0-1. However, since the scaling factor is calculated and stored at
-a certain time point, the symptom popularity released after that time point is
-likely to exceed the previously-observed maximum value which results in values
-larger than 1.
-
+## Properties and limitations 
+WTo learn more about the dataset, how it is generated it and its limitations,
```
Comment (dfarrow0) [inline docs/api/covidcast-signals/google-symptoms.md:53]: quick driveby: `WTo` -> `To`

File: docs/api/covidcast-signals/google-symptoms.md [hunk 2]
```diff
@@ -81,6 +68,7 @@ we are not provided).
 
 
 ## Lag and Backfill
-Google does not update the search data daily, but has an uncertain update
-frequency. The delay can range from 1 day to 10 days or even more. We check for
+Google does not currently update the search data daily, but usually twice a week.
+Each update will usually the coverage to within three days of the day of the update.
```
Comment (dfarrow0) [inline docs/api/covidcast-signals/google-symptoms.md:72]: might be missing a word here: "Each update will usually the coverage". maybe "extend the coverage"?

File: docs/api/covidcast-signals/google-symptoms.md [hunk 3]
```diff
@@ -1,39 +1,39 @@
...
+title: Google Search Trends symptoms dataset
 parent: Data Sources and Signals
 grand_parent: COVIDcast Epidata API
 ---
 
 # Google Symptoms
 {: .no_toc}
 
-* **Source name:** `google-symptoms`
+* **Source name:** `google-search-trends-symptoms-dataset`
```
Comment (dfarrow0) [inline docs/api/covidcast-signals/google-symptoms.md:10]: this is a technical identifier in the API, used in communication from clients to the server (as opposed to e.g. a descriptive label). changing this would break many things.

File: docs/api/covidcast-signals/google-symptoms.md [hunk 4]
```diff
@@ -1,39 +1,39 @@
...
 * **First issued:** 30 November 2020
 * **Number of data revisions since 19 May 2020:** 0
 * **Date of last change:** Never
 * **Available for:** county, MSA, HRR, state (see [geography coding docs](../covidcast_geography.md))
 * **Time type:** day (see [date format docs](../covidcast_times.md))
-* **License:** [CC BY](../covidcast_licensing.md#creative-commons-attribution)
+* **License:** To download or use the data, you must agree to the Google [Terms of Service](https://policies.google.com/terms)
```
Comment (dfarrow0) [inline docs/api/covidcast-signals/google-symptoms.md:16]: @capnrefsmmat please review license change

**Review by dfarrow0 — CHANGES_REQUESTED** [review]
thanks for the updates! generally looks good to me. i left a few comments inline.

two main things:
1. the source name `google-symptoms` can't easily be changed
2. requesting @capnrefsmmat to review the change to licensing

File: docs/api/covidcast-signals/google-symptoms.md (hunk 3 above)
Comment (shekelto) [reply docs/api/covidcast-signals/google-symptoms.md:10]: So please feel free to keep it as is.

File: docs/api/covidcast-signals/google-symptoms.md (hunk 2 above)
Comment (shekelto) [reply docs/api/covidcast-signals/google-symptoms.md]: bring or extend would be great

File: docs/api/covidcast-signals/google-symptoms.md (hunk 1 above)
Comment (shekelto) [reply docs/api/covidcast-signals/google-symptoms.md]: Fixed

File: docs/api/covidcast-signals/google-symptoms.md (hunk 3 above)
Comment (shekelto) [reply docs/api/covidcast-signals/google-symptoms.md]: I believe I submitted an updated request

File: docs/api/covidcast-signals/google-symptoms.md (hunk 4 above)
Comment (capnrefsmmat) [reply docs/api/covidcast-signals/google-symptoms.md:16]: The Google ToS are the terms listed in their GitHub repo for this data, so CC BY was never accurate. @shekelto is a Googler and tells us this is the right licensing to specify.

We do need to update `covidcast_licensing.md` to put this data into the "Republished Data" category, though

Comment (dfarrow0) [reply docs/api/covidcast-signals/google-symptoms.md:16]: ok great, thanks for taking a look

File: docs/api/covidcast-signals/google-symptoms.md (hunk 3 above)
Comment (dfarrow0) [reply docs/api/covidcast-signals/google-symptoms.md]: sounds good, thanks

**Review by dfarrow0 — APPROVED** [review]
lgtm

File: docs/api/covidcast-signals/google-symptoms.md [hunk 5]
```diff
@@ -49,22 +49,9 @@ The `sum_anosmia_ageusia_raw_search` signals are simply the raw sum of the
...
-over all symptoms. The values of symptom popularity are **NOT** comparable
-across geographic regions. Due to the scaling step, most of the values should be
-in the range 0-1. However, since the scaling factor is calculated and stored at
-a certain time point, the symptom popularity released after that time point is
-likely to exceed the previously-observed maximum value which results in values
-larger than 1.
-
+## Properties and limitations 
+To learn more about the dataset, how it is generated it and its limitations, 
+read the [dataset documentation](https://storage.googleapis.com/gcp-public-data-symptom-search/COVID-19%20Search%20Trends%20symptoms%20dataset%20documentation%20.pdf).
```
Comment (krivard) [inline docs/api/covidcast-signals/google-symptoms.md:54]: It's good to link out for more details, but I don't like the idea of hiding the region non-comparability limitation in page 3 of an external link. We can put a summary here for responsible caveat emptor purposes and let people who need the specifics click out.

```suggestion
When daily volume in a region does not meet quality or privacy thresholds, set
by Google, no value will be reported.

Google uses differential privacy, which adds artificial noise to the raw
datasets to avoid identifying any individual persons without affecting the
quality of results.

Google normalizes and scales time series values to determine the relative
popularity of symptoms in searches within each geographical region individually.
This means that the resulting values of symptom popularity are **NOT**
comparable across geographic regions. 

More details about the limitations of this dataset are available in [Google's Search 
Trends symptoms dataset documentation](https://storage.googleapis.com/gcp-public-data-symptom-search/COVID-19%20Search%20Trends%20symptoms%20dataset%20documentation%20.pdf).
```

File: docs/api/covidcast-signals/google-symptoms.md [hunk 6]
```diff
@@ -81,6 +68,7 @@ we are not provided).
 
 
 ## Lag and Backfill
-Google does not update the search data daily, but has an uncertain update
-frequency. The delay can range from 1 day to 10 days or even more. We check for
+Google does not currently update the search data daily, but usually twice a week.
+Each update will usually extend the coverage to within three days of the day of the update.
+As a result the delay can range from 3 to 10 days or even more. We check for
 updates every day and provide the most up-to-date data.
```
Comment (krivard) [inline docs/api/covidcast-signals/google-symptoms.md:74]: This section is listed in the template, but never made it into the page I guess? It would be nice to include a summary of what people are allowed to do with the data (who do they have to cite? can they sell anything they make using the data? do they have to share what they make under the same terms?) but not strictly necessary. 

Essentially we want people to be able to use this page to quickly evaluate whether this dataset is likely meet their needs, so that they can then decide whether it justifies reading a bunch of legal text.

```suggestion
updates every day and provide the most up-to-date data.

## Source and Licensing
This dataset is based on Google's [COVID-19 Search Trends symptoms dataset](http://goo.gle/covid19symptomdataset), which is licensed under Google's [Terms of Service](https://policies.google.com/terms).

To learn more about the source data, how it is generated and its limitations, 
read [Google's Search Trends symptoms dataset documentation](https://storage.googleapis.com/gcp-public-data-symptom-search/COVID-19%20Search%20Trends%20symptoms%20dataset%20documentation%20.pdf).
```

File: docs/api/covidcast-signals/google-symptoms.md [hunk 7]
```diff
@@ -13,27 +13,27 @@ grand_parent: COVIDcast Epidata API
 * **Date of last change:** Never
 * **Available for:** county, MSA, HRR, state (see [geography coding docs](../covidcast_geography.md))
 * **Time type:** day (see [date format docs](../covidcast_times.md))
-* **License:** [CC BY](../covidcast_licensing.md#creative-commons-attribution)
+* **License:** To download or use the data, you must agree to the Google [Terms of Service](https://policies.google.com/terms)
```
Comment (krivard) [inline docs/api/covidcast-signals/google-symptoms.md:16]: Is this text acceptable? we try to keep the header short, but if the extra language is required, we can make an exception.

```suggestion
* **License:** Google's [Terms of Service](https://policies.google.com/terms)
```

**Review by krivard — CHANGES_REQUESTED** [review]
Avoid hiding critical limitations; style conformance

File: docs/api/covidcast-signals/google-symptoms.md (hunk 7 above)
Comment (shekelto) [reply docs/api/covidcast-signals/google-symptoms.md:16]: Unfortunately it is required. We kept consistent messaging on the license across all of our surfaces.

File: docs/api/covidcast-signals/google-symptoms.md (hunk 5 above)
Comment (shekelto) [reply docs/api/covidcast-signals/google-symptoms.md:54]: I'm ok with this proposal and most of the text, but notice that when daily data is not available we are offering weekly data so perhaps it can be adjusted to be either:
"When daily volume in a region does not meet quality or privacy thresholds, set by Google, no daily value will be reported."
Or better:
"When daily volume in a region does not meet quality or privacy thresholds, set by Google, data will be provided at the weekly resolution. If the quality or privacy thresholds are not met at the weekly resolution, no data will be reported for the symptom in that region."

Comment (krivard) [reply docs/api/covidcast-signals/google-symptoms.md:54]: Yeah, but it won't show up in the COVIDcast data even if it's available in Google's weekly dataset. How about:

```suggestion
When daily volume in a region does not meet quality or privacy thresholds, set
by Google, no daily value is reported. Weekly data may be available from Google 
in these cases, but we do not yet support importation using weekly data.

Google uses differential privacy, which adds artificial noise to the raw
datasets to avoid identifying any individual persons without affecting the
quality of results.

Google normalizes and scales time series values to determine the relative
popularity of symptoms in searches within each geographical region individually.
This means that the resulting values of symptom popularity are **NOT**
comparable across geographic regions. 
More details about the limitations of this dataset are available in [Google's Search 
Trends symptoms dataset documentation](https://storage.googleapis.com/gcp-public-data-symptom-search/COVID-19%20Search%20Trends%20symptoms%20dataset%20documentation%20.pdf).
```

Comment (shekelto) [reply docs/api/covidcast-signals/google-symptoms.md:54]: Sounds good. 
Please hyperlink the words "available from Google" to https://pair-code.github.io/covid19_symptom_dataset.

**Review by krivard — APPROVED** [review]
Thank you for your patience 👍
